### ISSUE TITLE
solo.getCurrentWebElements() or solo.getWebElements() freeze the webview on screen

### ISSUE THREAD

**Issue by sinaweibo** [RobotiumTech/robotium#835]
i found the webview on the current screen frozen after i invoked solo.getCurrentWebElements() or solo.getWebElements(),even i touch the webview with my finger,it does not jump to a certain url。
robotium-solo-5.5.4.jar
android:4.4 and 5.1，may be all

**Comment by renas** [comment]
Would it be possible for you to share an APK that exhibits this issue?

**Comment by sinaweibo** [comment]
yes,but the apk is lagger than 10MB,i can't upload,would you give me a mailbox address that i can send the apk to you
